app/controllers/miq_request_controller.rb

```diff
@@ -585,22 +588,17 @@ def request_edit_settings(provision_request)
   end
 
   def get_session_data
+    super
     @title        = _("Requests")
     @request_tab  = session[:request_tab] if session[:request_tab]
-    @layout       = layout_from_tab_name(@request_tab)
-    @lastaction   = session[:request_lastaction]
     @showtype     = session[:request_lastaction]
-    @display      = session[:request_display]
     @current_page = session[:request_current_page]
```

Comment:
You can actually just skip `@current_page` - remove it from `get_session_data` and `set_session_data` and *not* add them to the mixin.

That's because `get_global_session_data` and `set_global_session_data` already handle `@current_page` ([set](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller.rb#L2181) and [get](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller.rb#L1948)). (Then, `@sb` is saved in the session under `session[:sandboxes][controller_name]`.)